src/models/whatsnewmodel.cpp

```diff
@@ -0,0 +1,148 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "whatsnewmodel.h"
+#include "leakdetector.h"
+#include "logger.h"
+#include "mozillavpn.h"
+#include "settingsholder.h"
+#include "featurelist.h"
+#include "feature.h"
+
+#include <QList>
+
+namespace {
+Logger logger(LOG_MODEL, "WhatsNewModel");
+}  // namespace
+
+WhatsNewModel::WhatsNewModel() { MVPN_COUNT_CTOR(WhatsNewModel); }
+
+WhatsNewModel::~WhatsNewModel() { MVPN_COUNT_DTOR(WhatsNewModel); }
+
+QList<Feature*> m_featurelist;
+bool m_initialized = false;
```

Comment:
Remove these two. They must be private members of the class. I suspect the issue is related to const methods. I'll comment them.